docs/eventlisteners.md

```diff
@@ -27,47 +92,60 @@ rules:
   verbs: ["create"]
 ```
 
-Note that currently, JSON is the only accepted MIME type for events.
+If your EventListener is using
+[`ClusterTriggerBindings`](./clustertriggerbindings.md), you'll need a
+ServiceAccount with a
+[ClusterRole instead](../examples/role-resources/clustertriggerbinding-roles/clusterrole.yaml)
+
+### ServiceType
+
+The serviceType field is optional. EventListener sinks are exposed via
```

Comment:
nit: could you please make `serviceType` a code snippet?

Reply:
👍